www-servers/nginx/patches/nginx-1.19.8.patchset

```diff
@@ -33,34 +33,34 @@ index d884487..fb487d7 100644
  
  
 diff --git a/auto/options b/auto/options
-index 521c976..b695ba6 100644
+index 80be906..8aac0bc 100644
 --- a/auto/options
 +++ b/auto/options
 @@ -13,6 +13,7 @@ NGX_CONF_PATH=
  NGX_ERROR_LOG_PATH=
  NGX_PID_PATH=
  NGX_LOCK_PATH=
-+NGX_DOC_PREFIX=
++NGX_DOC_PATH=
```

Comment:
This doesn't reflect the $value used in --doc-prefix?

Reply:
Oh wow, not even sure how that happened. I'll fix in a few.